tests/framework/installer/ceph_installer.go

```diff
@@ -207,71 +214,11 @@ func (h *CephInstaller) CreateK8sRookClusterWithHostPathAndDevices(namespace, sy
 		return fmt.Errorf("Failed to create cluster roles. %+v", err)
 	}
 
-	// for increasing the test matrix, sometimes use the strongly typed client instead of the yaml
-	if h.k8shelper.VersionAtLeast("v1.10.0") {
-		logger.Infof("Starting Rook cluster with strongly typed clientset")
-
-		clust := &cephv1beta1.Cluster{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      namespace,
-				Namespace: namespace,
-			},
-			Spec: cephv1beta1.ClusterSpec{
-				ServiceAccount:  "rook-ceph-cluster",
-				DataDirHostPath: dataDirHostPath,
-				Mon: cephv1beta1.MonSpec{
-					Count:                mon.Count,
-					AllowMultiplePerNode: mon.AllowMultiplePerNode,
-				},
-				Storage: rookalpha.StorageScopeSpec{
-					UseAllNodes: startWithAllNodes,
-					Selection: rookalpha.Selection{
-						UseAllDevices: &useAllDevices,
-					},
-					Config: map[string]string{
-						config.StoreTypeKey:      storeType,
-						config.DatabaseSizeMBKey: "1024",
-						config.JournalSizeMBKey:  "1024",
-					},
-				},
-			},
-		}
-		_, err := h.k8shelper.RookClientset.CephV1beta1().Clusters(namespace).Create(clust)
-		if err != nil {
-			return fmt.Errorf("failed to create cluster %s. %+v", clust.Name, err)
-		}
-
-		if !startWithAllNodes {
-			// now that the cluster is created, let's get all the k8s nodes so we can update the cluster CRD with them
-			logger.Info("cluster was started without all nodes, will update cluster to add nodes now.")
-			nodeNames, err := h.GetNodeHostnames()
-			if err != nil {
-				return fmt.Errorf("failed to get k8s nodes to add to cluster CRD: %+v", err)
-			}
-
-			// add all discovered k8s nodes to the cluster CRD
-			rookNodes := make([]rookalpha.Node, len(nodeNames))
-			for i, hostname := range nodeNames {
-				rookNodes[i] = rookalpha.Node{Name: hostname}
-			}
-			clust, err = h.k8shelper.RookClientset.CephV1beta1().Clusters(namespace).Get(namespace, metav1.GetOptions{})
-			if err != nil {
-				return fmt.Errorf("failed to get rook cluster to add nodes to it: %+v", err)
-			}
-			clust.Spec.Storage.Nodes = rookNodes
-
-			// update the cluster CRD now
-			_, err = h.k8shelper.RookClientset.CephV1beta1().Clusters(namespace).Update(clust)
-			if err != nil {
-				return fmt.Errorf("failed to update cluster %s with nodes. %+v", clust.Name, err)
-			}
-		}
-	} else {
-		logger.Infof("Starting Rook Cluster with yaml")
-		rookCluster := h.Manifests.GetRookCluster(&ClusterSettings{namespace, storeType, dataDirHostPath, useAllDevices, mon.Count})
-		if _, err := h.k8shelper.KubectlWithStdin(rookCluster, createFromStdinArgs...); err != nil {
-			return fmt.Errorf("Failed to create rook cluster : %v ", err)
-		}
+	logger.Infof("Starting Rook Cluster with yaml")
```

Comment:
I think there is/was an issue somewhere to use the yamls for testing, does this change cover the Ceph part of this?

Reply:
Now the ceph tests only test with the yaml since they are simpler to maintain than the strong types. I don't recall an issue around this and don't see a related issue, at least not with the `test` label, but let me know if you find it.